docker/scripts/ld.command.git-repo-massage.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+# File
+#
+# This file contains git-repo-massage -command for local-docker script ld.sh.
+
+function ld_command_git-repo-massage_exec() {
+
+    if [ ! -d ".git" ]; then
+        echo -e "${Red}ERROR: Could not locate Git repository (.git folder missing).${Color_Off}"
+        return 1
+    fi
+
+    # Check if we have initial local-docker commit in git history.
+    GIT_COMMIT_MATCH=$(git rev-list HEAD | tail -n 1 |grep -c "99c1a03ce16f2ff1fe33bc136e09fcbcb0fd5397" )
+    [ "$LD_VERBOSE" -ge "2" ] && [ "$GIT_COMMIT_MATCH" -gt "0" ] && echo "Initial Git commit matches known initial commit hash in Exove/local-docker.git"
+
+    # This catches both https and ssh based remote addresses.
+    GIT_REPO_MATCH=$(git config --get remote.origin.url | grep -c 'Exove/local-docker.git')
+
+    [ "$LD_VERBOSE" -ge "2" ] && [ "$GIT_REPO_MATCH" -gt "0" ] && echo "Git repository is connected to Exove/local-docker.git"
+
+    if [ "$GIT_REPO_MATCH" -gt "0" ]; then
+        echo -e "${BYellow}NOTE${Yellow}: You are still connected to ${BYellow}Exove/local-docker.git${Yellow}.${Color_Off}"
+        VALID=
+        echo
+        echo -e "${BBlack}== GIT repository address ==${Color_Off}"
+        while [ -z "$VALID" ]; do
+            echo "What is your Git repository address?"
+            echo "It is YOUR responsibility to ensure it is correct and functional. However local-docker does not touch your git remote, ever."
+            echo "If you leave this empty Git remote will not be edited."
+            read -p "Git repository address: " ANSWER
+#            # Very simple test to see we have .git at the end of remote url.
```

Comment:
Two comment markers, for any specific purpose?
```suggestion
             # Very simple test to see we have .git at the end of remote url.
```